Marlin/src/module/tool_change.cpp

```diff
@@ -274,31 +397,16 @@
                 grabxpos = toolheadposx[tmp_extruder];
 
     /**
-     * 1. Raise Z to give enough clearance
-     * 2. Move to switch position of current toolhead
-     * 3. Unlock tool and drop it in the dock
-     * 4. Move to the new toolhead
-     * 5. Grab and lock the new toolhead
-     * 6. Apply the z-offset of the new toolhead
+	 * Z Lift and Nozzle Offset shift ar defined in caller method to work equal with any Multi Hotend realization
+	 *
+     * 1. Move to switch position of current toolhead
+     * 2. Unlock tool and drop it in the dock
+     * 3. Move to the new toolhead
+     * 4. Grab and lock the new toolhead
      */
 
     // STEP 1
 
-    #if ENABLED(DEBUG_LEVELING_FEATURE)
-      if (DEBUGGING(LEVELING)) DEBUG_POS("Starting Toolhead change", current_position);
-    #endif
-
-    current_position[Z_AXIS] += toolchange_settings.z_raise;
-
-    #if ENABLED(DEBUG_LEVELING_FEATURE)
-      if (DEBUGGING(LEVELING)) DEBUG_POS("(1) Raise Z-Axis", current_position);
-    #endif
-
-    planner.buffer_line(current_position, planner.settings.max_feedrate_mm_s[Z_AXIS], active_extruder);
-    planner.synchronize();
-
-    // STEP 2
-
```

Comment:
Why is the normal solenoid-based `PARKING_EXTRUDER` code being altered in this PR?

Reply:
Same Issue as in Magnetic Parking Extruder. Z-Axis setup twice. also from caller method, because MPE was based from Parking Extruder. 
I'll revert this change and open a separate issue with pullrequest if it was accepted from the issue discussion.